game/scripts/npc/items/item_pipe.txt

```diff
@@ -63,8 +63,8 @@
     {
       "01"
       {
-        "var_type"                                        "FIELD_INTEGER"
-        "health_regen"                                    "8 12 18 27 40"
+        "var_type"                                        "FIELD_INTEGER" // 4 6 9 13 to 3.1 4.8 7.2 11.4
```

Comment:
do not quite understand the point of this comment

Reply:
just doing math, first set is the difference between levels before, second is after. i do it a few times